lib/ng-bank-parser.rb

```diff
@@ -4,8 +4,11 @@
 require "ng-bank-parser/pdf-unlocker"
 require "ng-bank-parser/parsers/gtb-excel-parser"
 require "ng-bank-parser/parsers/uba-pdf-parser"
-require "ng-bank-parser/parsers/firstbank-pdf-parser"
+require "ng-bank-parser/parsers/accessbank-pdf-parser"
 
 module NgBankParser
   # Your code goes here...
+
+  AccessbankPdf.parse('lib/ng-bank-parser/fixtures/accessbank-pdf-valid.pdf')
```

Comment:
No code should go in the main parser block. You should only require your parser in this file